src/connection.js

```diff
@@ -882,12 +952,19 @@ export class Connection {
     }
     // End Legacy v0.16 response
 
-    const res = GetRecentBlockhash(unsafeRes);
+    const res = GetRecentBlockhashWithCommitmentRpcResult(unsafeRes);
     if (res.error) {
       throw new Error(res.error.message);
     }
     assert(typeof res.result !== 'undefined');
-    return res.result;
+    return RpcResponse(res.result);
+  }
+  async getRecentBlockhash(
```

Comment:
Do we really need to have two separate methods? `x` and `xWithCommitment`? I think it would be better to stick with one and break the previous api

Reply:
I like this sentiment, will go for the dispatch idea